yarn-audit-known-issues

```diff
@@ -1,25 +1,2 @@
-{"type":"auditAdvisory","data":{"resolution":{"id":1006724,"path":"@hmcts/cmc-draft-store-middleware>@hmcts/draft-store-client>request>http-signature>jsprim>json-schema","dev":false,"optional":false,"bundled":false},"advisory":{"findings":[{"version":"0.2.3","paths":["request>http-signature>jsprim>json-schema","@hmcts/draft-store-client>request>http-signature>jsprim>json-schema","@hmcts/cmc-draft-store-middleware>@hmcts/draft-store-client>request>http-signature>jsprim>json-schema"]}],"metadata":null,"vulnerable_versions":"<0.4.0","module_name":"json-schema","severity":"moderate","github_advisory_id":"GHSA-896r-f27r-55mw","cves":["CVE-2021-3918"],"access":"public","patched_versions":">=0.4.0","updated":"2021-11-15T22:44:28.000Z","recommendation":"Upgrade to version 0.4.0 or later","cwe":"CWE-915","found_by":null,"deleted":null,"id":1006724,"references":"- https://nvd.nist.gov/vuln/detail/CVE-2021-3918\n- https://github.com/kriszyp/json-schema/commit/22f146111f541d9737e832823699ad3528ca7741\n- https://huntr.dev/bounties/bb6ccd63-f505-4e3a-b55f-cd2662c261a9\n- https://github.com/kriszyp/json-schema/commit/b62f1da1ff5442f23443d6be6a92d00e65cba93a\n- https://github.com/kriszyp/json-schema/commit/f6f6a3b02d667aa4ba2d5d50cc19208c4462abfa\n- https://github.com/advisories/GHSA-896r-f27r-55mw","created":"2021-11-19T21:00:41.916Z","reported_by":null,"title":"json-schema is vulnerable to Prototype Pollution","npm_advisory_id":null,"overview":"json-schema is vulnerable to Improperly Controlled Modification of Object Prototype Attributes ('Prototype Pollution')","url":"https://github.com/advisories/GHSA-896r-f27r-55mw"}}}
 {"type":"auditAdvisory","data":{"resolution":{"id":1004876,"path":"@hmcts/class-validator>validator","dev":false,"optional":false,"bundled":false},"advisory":{"findings":[{"version":"10.4.0","paths":["@hmcts/class-validator>validator","@hmcts/cmc-validators>@hmcts/class-validator>validator"]}],"metadata":null,"vulnerable_versions":"<13.7.0","module_name":"validator","severity":"moderate","github_advisory_id":"GHSA-qgmg-gppg-76g5","cves":["CVE-2021-3765"],"access":"public","patched_versions":">=13.7.0","updated":"2021-11-03T14:46:01.000Z","recommendation":"Upgrade to version 13.7.0 or later","cwe":"CWE-1333","found_by":null,"deleted":null,"id":1004876,"references":"- https://nvd.nist.gov/vuln/detail/CVE-2021-3765\n- https://github.com/validatorjs/validator.js/commit/496fc8b2a7f5997acaaec33cc44d0b8dba5fb5e1\n- https://huntr.dev/bounties/c37e975c-21a3-4c5f-9b57-04d63b28cfc9\n- https://github.com/advisories/GHSA-qgmg-gppg-76g5","created":"2021-11-18T16:00:48.460Z","reported_by":null,"title":"Inefficient Regular Expression Complexity in validator.js","npm_advisory_id":null,"overview":"validator.js prior to 13.7.0 is vulnerable to Inefficient Regular Expression Complexity","url":"https://github.com/advisories/GHSA-qgmg-gppg-76g5"}}}
```

Comment:
Is the stuff removed not important?

Reply:
This was a security update a lot of the issues are now resolved.